vue-frontend/src/components/home-new/ServiceSection.vue

```diff
@@ -0,0 +1,119 @@
+<template>
+  <div class="tw-items-left tw-justify-left tw-pt-[50px] md:tw-pt-[100px]">
+    <div
+      class="tw-flex tw-justify-center sm:tw-py-12 tw-py-9 tw-font-roboto-medium tw-leading-[4.46875rem] sm:tw-text-[3.4375rem] tw-text-[2.188rem]"
+    >
+      How We Can Help You
+    </div>
+
+    <div
+      v-for="(service, index) in services"
+      :key="service"
+      @mouseover="data = index"
+      @mouseleave="data = null"
+      class="tw-relative tw-text-[1.1rem] sm:tw-text-[1.5625rem] md:tw-h-28 lg:tw-h-28 tw-w-full"
+    >
+      <div class="">
+        <div
+          class="service-title tw-flex md:tw-text-[1.2rem] lg:tw-text-[1.5625rem] tw-ml-0 tw-absolute tw-left-0 tw-top-0 tw-flex md:tw-flex-row tw-flex-col"
+        >
+          <div
+            class="md:tw-h-28 tw-basis-2/4 tw-flex tw-pl-[1rem] xl:tw-pl-[4.563rem] 2xl:tw-pl-[16.563rem] tw-justify-right tw-items-center"
+          >
+            <span
+              class="sm:tw-py-9 tw-py-3 tw-font-sfp-regular tw-flex tw-flex-row"
+            >
+              {{ service.series_no }}
+            </span>
+            <span
+              class="sm:tw-p-9 tw-p-3 tw-font-sfp-regular tw-flex tw-flex-row"
+            >
+              {{ service.title }}
```

Comment:
Text is different. you can match with figma's text

Reply:
I have used Roboto only.